### ISSUE TITLE
About jaggies and green edges after keying.

### ISSUE THREAD

**Issue by royaiagent** [GeekyGhost/ComfyUI-GeekyRemB#4]
The green screen video is either jagged or has a green edge after keying the portrait, and the composited video is still poor on the edges as well
<img width="441" alt="image" src="https://github.com/user-attachments/assets/76ace00f-b314-456f-bcd2-02d060876ed9">
<img width="558" alt="image" src="https://github.com/user-attachments/assets/832497f4-8273-44f1-8f25-87c8bf6e280b">

**Comment by GeekyGhost** [comment]
Yeah, you have to adjust the settings based on the image. Alpha matting, chroma key, mask expansion and shrinkage, you can also use additional masks to increase quality. Right now the focus has shifted to the animation functions, but I have some updates for the removal coming soon as well. Just have to make sure I don't break the layer stacking or it messes everything else .

So for the time being, play with the settings and you'll get there. Experiment with different models also, some work better than others for various images. But an update to do a lot of the adjustments automatically using a vision model is coming. I will not elaborate at this time 😅.

**Comment by royaiagent** [comment]
Thank you!    

I would also like to ask if the maximum value of animation_frames can be increased to 3000-6000 so that the user can compose longer videos.

**Comment by GeekyGhost** [comment]
That is doable, absolutely.

**Comment by GeekyGhost** [comment]
This should be resolved
